app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/model/SourceDefinition.scala

```diff
@@ -44,11 +44,11 @@ object SourceDefinition extends LazyLogging {
   ) {
     @SuppressWarnings(Array("OptionGet"))
     def toSourceDefinition: SourceDefinition = {
-      if (extent.isDefined && crs.isDefined && cellSize.isDefined) {
+      if (extent.isDefined && crs.isDefined && cellSize.isDefined && extentCrs.isDefined) {
```

Comment:
Would there be cases where a user would want to specify a larger extent for the tif than the tags indicate? If not I think that makes a lot of sense.